CMakeLists.txt

```diff
@@ -100,26 +80,43 @@ include_directories("${PROJECT_SOURCE_DIR}/CHI_TECH")
 include_directories("${PROJECT_SOURCE_DIR}/CHI_TECH/CHI_MATH/CHI_DISCRETIZATION")
 include_directories(SYSTEM ${MPI_INCLUDE_PATH})
 include_directories("${PETSC_ROOT}/include")
-include_directories("${BOOST_ROOT}/include")
+include_directories(SYSTEM "${BOOST_ROOT}/include")
 
-include_directories("${VTK_ROOT}/include/vtk${VTK_VERSION}")
 
 #================================================ Library directories
 link_directories("${LUA_ROOT}/lib")
 link_directories("${TRIANGLE_ROOT}")
 link_directories("${PETSC_ROOT}/lib")
 #link_directories("${BOOST_ROOT}/lib")
 
-link_directories("${VTK_ROOT}/lib")
-link_directories("${VTK_ROOT}/lib64")
-
 set(TRIANGLE "${TRIANGLE_ROOT}/triangle.o")
 
-set(LIBS lua m dl ${MPI_CXX_LIBRARIES} petsc)
-set(LIBS ${LIBS} vtkCommonCore${VTK_VERSION} vtkCommonDataModel${VTK_VERSION})
-set(LIBS ${LIBS} vtkIOLegacy${VTK_VERSION} vtkIOCore${VTK_VERSION})
-set(LIBS ${LIBS} vtkIOXML${VTK_VERSION} vtkParallelCore${VTK_VERSION} )
-set(LIBS ${LIBS} vtkIOParallelXML${VTK_VERSION})
+# --------------------------- VTK
+find_package(VTK COMPONENTS
```

Comment:
This seems to work fine on local machines with VTK installed. Just for #shits&giggles I tested your fork on quartz and then it fails without really any message on how to fix it.

If we add "REQUIRED": `find_package(VTK REQUIRED COMPONENTS` then add least cmake fails and tells the user to set `VTK_DIR` after which it works.

Reply:
Good point - I've pushed the update.